### ISSUE TITLE
Static Maps-Soil hydraulic properties maps

### ISSUE THREAD

**Issue by kferhat1** [ec-jrc/lisflood-code#96]
I calculated the soil hydraulic properties by using the formula and the pedotransfer functions via ''https://ec-jrc.github.io/lisflood-code/4_Static-Maps_soil-hydraulic-properties/ ''. However, my calculated ones and the already existing one (e.g. ksat values) are different from each other(even different as orders or magnitudes). Is there anyone who were faced with such kind of situations in calculating the soil hydraulic properties with respect to layers? Thank you for your answers.
#staticmaps #soilhydraulicproperties

**Comment by StefaniaGrimaldi** [comment]
Dear @kferhat1 , thanks for your question. Could you please clarify which are "the already existing" soil property maps?

**Comment by kferhat1** [comment]
Thank you for your quick response dear @StefaniaGrimaldi . Actually, I have already had genualpha, ksat, thetaresidual nc. files(genua1_f, genua2_o,ksat2_o... etc.) for the basin that has been choosen before, which are already existing maps. When I want to calculate the same basin by myself by using the proceudre in ''https://ec-jrc.github.io/lisflood-code/4_Static-Maps_soil-hydraulic-properties/ '' adress, I could not obtain the same results. P.S.: both already existing and newly calculated(by me) have the same resolution, same projection. How can I fix it or obtain the same results? Thank you

**Comment by StefaniaGrimaldi** [comment]
Dear @kferhat1, thanks for providing more details. 
If my understanding is correct, you are comparing your own results with the soil properties maps of the Fraser catchment and of the Po basin. More specifically, the  soil properties maps of the Fraser and of the Po that are available either from the repository lisflood-usecases https://github.com/ec-jrc/lisflood-usecases or the test folder of the repository lisflood-code https://github.com/ec-jrc/lisflood-code/tree/master/tests/data. 
Is my understanding correct? If so, it is OK to see large differences between your own results and those maps. 
The maps of the above mentioned repositories were produced using the previous (now outdated) protocol (https://ec-jrc.github.io/lisflood/pdfs/Dataset_hydro.pdf). 
We very recently revised the entire protocol to produce the static maps: https://ec-jrc.github.io/lisflood-code/4_Static-Maps_soil-hydraulic-properties/ explains the updated methodology.
We are aware that the updated methodology leads to very different (even different order of magnitudes) values for some specific soil properties and in some areas of the World. The updated methodology (https://ec-jrc.github.io/lisflood/pdfs/Dataset_hydro.pdf) was validated by comparing the results with a number of peer reviewed publications.
In summary, if you are comparing your own results with the maps of the repository lisflood-usecases https://github.com/ec-jrc/lisflood-usecases or the maps of the test folder of the repository lisflood-code https://github.com/ec-jrc/lisflood-code/tree/master/tests/data, then it is OK to see large differences between your own maps and the existing maps.
We missed to clarify this mismatch in the "read me" file of the user cases: we will take action and add a sentence in the next few days. 
We hope this explanation solves your issue.  
Kind regards, Stefania

**Comment by StefaniaGrimaldi** [comment]
The  "read me" file of the user cases has been updated "NOTE about the static maps made available by this repository": 
https://github.com/ec-jrc/lisflood-usecases
